arviz/plots/ppcplot.py

```diff
@@ -4,81 +4,104 @@
 from .plot_utils import _scale_text, _create_axes_grid, default_grid
 
 
-def ppcplot(data, ppc_sample, kind='kde', mean=True, figsize=None, textsize=None, ax=None):
+def ppcplot(data, kind='kde', alpha=0.2, mean=True, figsize=None, textsize=None):
     """
     Plot for Posterior Predictive checks.
 
+    Note that this plot will flatten out any dimensions in the posterior predictive variables.
+
     Parameters
     ----------
     data : Array-like
         Observed values
-    ppc_samples : dict
-        Posterior predictive check samples
     kind : str
         Type of plot to display (kde or cumulative)
+    alpha : float
+        Opacity of posterior predictive density curves
     mean : bool
-        Whether or not to plot the mean ppc distribution. Defaults to True
+        Whether or not to plot the mean posterior predictive distribution. Defaults to True
     figsize : figure size tuple
         If None, size is (6, 5)
     textsize: int
         Text size for labels. If None it will be auto-scaled based on figsize.
-    ax: axes
-        Matplotlib axes
 
     Returns
     -------
-    ax : matplotlib axes
+    axes : matplotlib axes
     """
-    rows, cols = default_grid(len(ppc_sample))
-
-    if figsize is None:
-        figsize = (7, 5)
+    for group in ('posterior_predictive', 'observed_data'):
+        if not hasattr(data, group):
+            raise TypeError(
+                '`data` argument must have the group "{group}" for ppcplot'.format(group=group))
 
-    _, ax = _create_axes_grid(len(ppc_sample), rows, cols, figsize=figsize)
+    observed = data.observed_data
```

Comment:
Why not use `convert_to_dataset` like the other plots?

Reply:
convert to dataset does not implement posterior and other extra values yet.

Reply:
To word this specifically, convert_to_dataset does not implement posterior *predictive* distributions yet. In my understanding it does supporting posterior parameters/trace conversely?